byterestclient/__init__.py

```diff
@@ -35,12 +35,19 @@ def request(self, method, path, data=None, *args, **kwargs):
             **kwargs
         )
 
-        response.raise_for_status(allow_redirects=False)
+        response.raise_for_status()
+
+        if 300 <= response.status_code < 400:
+            # support 0.12.1 by setting response seperately
+            exception = HTTPError('%s Encountered Redirect' % response.status_code)
+            exception.response = response
+            raise exception
```

Comment:
Yeah, there is `requests.models.REDIRECT_STATI`, which is faulty latin and
an ugly import :wink:.

But should probably use that.